build_scripts/webpack.config.js

```diff
@@ -103,6 +103,10 @@ module.exports = {
     host: '0.0.0.0',
     port: 8080,
     disableHostCheck: true,
-    hot: !helpers.isProduction
+    hot: !helpers.isProduction,
+    proxy: {
```

Comment:
proxy for now until we use params passed from Desktop and a potential CORS issue that was being experienced